runtime/packages/core-modules/assembly/system.ts

```diff
@@ -1,13 +1,19 @@
-import { CompactInt, ByteArray } from 'as-scale-codec';
+import { CompactInt, ByteArray, Bytes, UInt64 } from 'as-scale-codec';
 import { Storage } from './storage';
-import { Blocks, Header } from '@as-substrate/models';
-import { Utils } from '@as-substrate/core-utils';
+import { Blocks, Header, Signature } from '@as-substrate/models';
+import { Utils, Serialiser } from '@as-substrate/core-utils';
+import { AccountId } from '@as-substrate/balances-module';
+import { ext_crypto_sr25519_verify_version_2 } from './env';
 
 export class System {
     // execution phases
     static readonly APPLY_EXTRINSIC: string = "ApplyExtrinsic";
     static readonly INITIALIZATION: string = "Initialization";
-    static readonly FINALIZATION: string = "Finalization"
+    static readonly FINALIZATION: string = "Finalization";
+    // number of all modules in the runtime that creates inherents (timestamp and aura, for now)
+    static readonly ALL_MODULES: u8[] = [4];
```

Comment:
It could be confusing to see 4 but state that only 2 are producing inherents. I think that you should specify that `4` is Compact Int 2